custom-tests.json

```diff
@@ -205,232 +205,232 @@
       "__base": "if (!reusableInstances.audioContext) {return false;} var instance = reusableInstances.audioContext.createGain();"
     },
     "HTMLAnchorElement": {
```

Comment:
Can't we instead do `if (instance.constructor.name !== 'HTMLAnchorElement') { return false; }`? Or are the very old browsers where `instance.constructor.name` is the wrong thing?

Reply:
Taken care of in #644!